crates/pruning/src/tests.rs

```diff
@@ -0,0 +1,31 @@
+use std::thread::sleep;
+use std::time::Duration;
+
+use sov_db::ledger_db::LedgerDB;
+use sov_db::rocks_db_config::RocksdbConfig;
+use tokio::sync::broadcast;
+use tokio_util::sync::CancellationToken;
+
+use crate::{Pruner, PruningConfig};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_pruner_simple_run() {
```

Comment:
What does this test do?

Reply:
Run the pruner with a cleanup distance of 5 L2 blocks. Generate 10 blocks (runs the pruning twice, but we don't implement pruning right now so nothing happens).

And then when the cancellation token is invoked, the pruner should wait for existing tasks to stop and the test will pass.

This will be useful after we implement the actual pruning logic